app/views/exports/new.html.haml

```diff
@@ -1,3 +1,6 @@
 %h4 Create new export
+.pull-right
```

Comment:
I really think this needs to be a part of the buttons in the form.  I'm also thinking we need to revamp how they are laid out in the page.  Before when the export form was a modal popup it was a narrow width so the buttons were that far apart.  But now with it being it's own page they really should just be next to each other and not really using the pull-right stuff.